### PR TITLE
Test multiple Angular versions in Travis

### PR DESCRIPTION
**Pull request by henrahmagix**
@incuna/frontend Please merge, ta!

### CODE REVIEW

File: scripts/setBowerResolution.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Set bower resolutions for Angular packages to value of --angular-version.
+// Without a resolution, `bower install` will ask for one, which isn't possible
+// in Travis.
+
+if (!process.env.CI) {
+    process.exit(0);
+}
+
+'use strict';
```
Comment (wytrych) [inline scripts/setBowerResolution.js]: Shouldn't this be at the top?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+cache:
+  directories:
+    - '$HOME/.npm'
+    - '$HOME/.cache/bower'
+
+env:
+  - ANGULAR_VERSION=1.2.29
+  - ANGULAR_VERSION=1.3.20
+  - ANGULAR_VERSION=1.4.9
```
Comment (wytrych) [inline .travis.yml:16]: What about `1.5.1` ?

Comment (henrahmagix) [inline .travis.yml:16]: This package is pointless for 1.5.0+. I set the dependency range to exclude that. See the README

With the method that's used to install the specific version – see `scripts/setBowerResolution.js` – it's not possible to test the bower installation, so there's nothing to test for 1.5.0+

File: scripts/setBowerResolution.js (hunk 1 above)
Comment (henrahmagix) [inline scripts/setBowerResolution.js]: Done in 77c6664

**Comment by henrahmagix** [comment]
Thanks!

**Comment by henrahmagix** [comment]
Released as [v1.0.0](https://github.com/incuna/angular-touch-no-ng-click/releases/tag/v1.0.0)

**Comment by henrahmagix** [comment]
Registered in the bower registry

**Pull request merged**